share/templates/08-Orchestral/02-Symphony_Orchestra.mscx

```diff
@@ -92,6 +95,7 @@
         </section>
       <section id="brass" brackets="true" showSystemMarkings="false" barLineSpan="true" thinBrackets="true">
         <family>horns</family>
+        <family>flugelhorns</family>
```

Comment:
Same as above, make sure the score order here is similar as the same score order in <code>orders.xml</code>.